source/deployment-guide/server/air-gapped-deployment.rst

```diff
@@ -1,200 +1,170 @@
-      
-      Download from https://github.com/goharbor/harbor/releases
+.. tab:: Kubernetes
 
-   2. **Extract and configure**:
+   Kubernetes is recommended for a highly scalable and robust deployment if your organization is already running a Kubernetes cluster in the air-gapped environment.
 
-      .. code-block:: bash
+    **Prerequisites**
 
-         tar xzvf harbor-offline-installer-v2.x.x.tgz
-         cd harbor
-         cp harbor.yml.tmpl harbor.yml
-         # Edit harbor.yml to configure settings
+    -  `Mattermost Operator <https://github.com/mattermost/mattermost-helm/tree/master/charts/mattermost-operator>`_ and `values <https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-operator/values.yaml>`__
+    - Database: We recommend options such as the `Postgres Operator <https://access.crunchydata.com/documentation/postgres-operator/latest/quickstart>`_ from Crunchy Data, `CloudNativePG <https://cloudnative-pg.io/documentation/1.27/installation_upgrade/>`__ or `pgEdge <https://github.com/pgEdge/pgedge-helm>`__.
+    - File Storage: We recommend the `MinIO Operator <https://github.com/minio/operator>`__.
+    - Load balancer: If you already have a load balancer running in your air-gapped environment you can skip this resource, otherwise we recommend deploying :doc:`NGINX </deployment-guide/server/setup-nginx-proxy>`, using the `NGINX Ingress Controller operator <https://docs.nginx.com/nginx-ingress-controller/installation/installing-nic/installation-with-operator/>`__.
 
-   3. **Install Harbor**:
+    **(Optional) Supporting Services**
+    Consider downloading these additional resources if you plan to enable these optional components:
 
-      .. code-block:: bash
-
-         ./install.sh --with-trivy
+    - :doc:`Mattermost Calls </administration-guide/configure/calls-deployment>` helm charts: `mattermost-calls-offloader <https://github.com/mattermost/mattermost-helm/tree/master/charts/mattermost-calls-offloader>`__ and `values <https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-calls-offloader/values.yaml>`__ (required for recording, transcription and live captions), `mattermost-rtcd <https://github.com/mattermost/mattermost-helm/tree/master/charts/mattermost-rtcd>`__ and `values <https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-rtcd/values.yaml>`__ (required for performance and scalability).
+    - `Elasticsearch <https://www.elastic.co/docs/deploy-manage/deploy/cloud-on-k8s>`__ can be `deployed in air-gapped k8 environments <https://www.elastic.co/guide/en/cloud-on-k8s/2.8/k8s-air-gapped.html>`__ for enhanced search performance at scale.
+    - `Prometheus <https://github.com/prometheus-operator/prometheus-operator>`__ and `Grafana <https://github.com/grafana/grafana-operator>`__ operators for monitoring and observability
 
-   4. **Access Harbor** at ``https://harbor-hostname`` (based on your configuration)
+.. tab:: Docker
```

Comment:
should we remove this? it's unfortunate but wondering if it conflicts with our other suggestions.

Reply:
Is there a world where it is used for local development or non HA instances? I would recommend aligning this doc to our [server deploy doc](https://docs.mattermost.com/deployment-guide/server/server-deployment-planning.html#deployment-options), which still has docker an option. So until and unless we remove it from there, I don't think it should be omitted here. Thoughts? 

We state: "Docker can be used if you don't have a running Kubernetes cluster in the air-gapped environment, but want to use containers for simplified installation and dependency management. Docker is not recommended for production environments at high scale, as it doesn’t support clustered deployments or High Availability (HA) configurations out-of-the-box."

Reply:
I'm in support of docker, i only bring it up because internally it was decided to move away from it as an option. So, potentially unifying our stance would be good. but 0/5 on what we do here. The docker content in this PR is correct as far as i know.

Reply:
Thanks. I'd say we keep it in until and unless we remove it from the server deploy page. We state the drawbacks/considerations for high scale so I think we're covered here.